packages/storybook/src/nimble/button/button.mdx

```diff
@@ -1,45 +1,269 @@
 import { Canvas, Meta, Controls, Title } from '@storybook/addon-docs/blocks';
```

Comment:
In my browser (macOS Firefox) the page load time seems not great. 

Warm refresh of the production storybook is <1 second: https://nimble.ni.dev/storybook/?path=/docs/components-button--docs

But warm refresh of this page is 6-8 seconds: https://60e89457a987cf003efc0a5b-mgbjyxprll.chromatic.com/?path=/docs/components-button--docs

It's possible this is due to the chromatic URL / pre-deployment build but I'm worried the page might be heavier than before.